eosclubhouse/quests/hack2/onboardtour.py

```diff
@@ -0,0 +1,352 @@
+        self.onboarding_clean()
+
+        self.wait_confirm('SETTINGS_MOVEWIN')
+
+        self.show_highlight_widget('panel-status-indicators-box user-mode-indicators-box')
+        if self.has_connection():
+            self.wait_confirm('SETTINGS_HASNET')
+        else:
+            self.wait_confirm('SETTINGS_NONET')
+
+        self.wait_confirm('SETTINGS_TRAY_VOL')
+
+        if self.comp_islaptop:
+            self.wait_confirm('SETTINGS_LAP_BATT')
+        self.wait_confirm('SETTINGS_TRAY_INFO')
+
+        self.show_highlight_widget('panelRight')
+        self.wait_confirm('SETTINGS8')
+
+        # TODO: It'd be cool if we could highlight the
+        # 'settings' button in the avatar menu here
+        self.wait_confirm('SETTINGS9')
+        # TODO: For some reason this doesn't work with the ControlCenter
+        # self.show_hints_message(message_id='SETTINGS9')
+        # self.wait_for_app_in_foreground(app=self.settingsApp)
+        self.onboarding_clean()
+
+        self.wait_confirm('SETTINGS10')
+        # TODO: Can we highlight single/multiple entries
+        # in Settings using clippy?
+        self.wait_confirm('SETTINGS11')
+        # TODO: clippy highlight WiFi
+        self.wait_confirm('SETTINGS12')
+        # TODO: clippy highlight Background
+        self.wait_confirm('SETTINGS13')
+        # TODO: clippy highlight Power
+        self.wait_confirm('SETTINGS14')
+
+        if self.comp_islaptop:
+            self.wait_confirm('SETTINGS_LAP_PWR')
+        return self.step_updates
+
+    def step_updates(self):
+        self.wait_confirm('UPDATES1')
+        self.wait_confirm('UPDATES2')
+        self.wait_confirm('UPDATES3')
+        self.wait_confirm('UPDATES4')
+        # TODO: clippy highlight Automatic Updates
+        self.wait_confirm('UPDATES5')
+        return self.step_morethings
+
+    def step_morethings(self):
+        askmore_action = self.show_choices_message('Q_KNOWMORE',
+                                                   ('A_SOMEMORE', None, 'more'),
+                                                   ('A_NOMORE', None, 'no')).wait()
+        # if askmore_action.future.result() == 'hack':
+        #     self.wait_confirm('WHATSHACK')
+        #     return self.step_morethings
+        # if askmore_action.future.result() == 'how':
+        #     # TODO: if we want this, it will be broken out into a much bigger section
+        #     self.wait_confirm('HOWDOI')
+        #     return self.step_morethings
+        # if askmore_action.future.result() == 'progress':
+        #     return self.step_ending
+        if askmore_action.future.result() == 'more':
+            self.wait_confirm("MORE_HACK1")
+            self.wait_confirm("MORE_HACK2")
+        return self.step_ending
+
+    def step_ending(self):
+        self.wait_confirm('ENDING1')
+        self.wait_confirm('ENDING3')
+        self.wait_confirm('ENDING4')
+        self.onboarding_clean()
+        super().step_complete_and_stop()
```

Comment:
You should just `return self.step_complete_and_stop`

Reply:
Ok, done.